src/site/content/en/learn/html/headings-and-sections/index.md

```diff
@@ -0,0 +1,209 @@
+
+{% Img src="image/kheDArv5csY6rvQUJDbWRscckLr1/A6uTEfG3muxX7namzXW5.png", alt="The Accessibility Object Model in Chrome.", width="800", height="374" %}
+
+In this screenshot, there are two footers: one in an `<article>` and one top level. The top level footer is a landmark with the implicit role of `contentinfo`. The other footer isn't a landmark. Chrome shows it as `FooterAsNonLandmark`; Firefox shows it as `section`.
+
+That doesn't mean you shouldn't use `<footer>`. Let's say you have a blog. The blog has a site footer with an implicit `contentinfo` role. Each blog post can also have a `<footer>`. On your blog's main landing page, the browser, search engine, and screen reader know the main footer is the top-level footer, and that all the other footers are related to the posts in which they are nested. 
+
+When a `<footer>` is a descendant of an `<article>`, `<aside>`, `<main>`, `<nav>`, or `<section>`, it's not a landmark. If the post appears on its own, depending on the markup, that footer might get promoted. 
+
+Footers are often where you will find contact information, wrapped in `<address>`, the contact address element. This is one element that is not very well named; it is used to enclose the contact information for individuals or organizations, not physical mailing addresses.
+
+```html
+<footer>    
+  <p>&copy;2022 Machine Learning Workshop, LLC. All rights reserved.</p>
+  <address>Instructors: <a href="/hal.html">Hal</a> and <a href="/eve.html">Eve</a></address>
+</footer>
+```
+
+### Document structure
+
+This module starts with the `<header>` and `<footer>`, because they are unique in only sometimes being landmark, or "sectioning", elements. Let's cover the "full time" sectioning element by discussing the most common page layouts:
+
+{% Img src="image/kheDArv5csY6rvQUJDbWRscckLr1/LJs8vlTVDYBmM6t1z6rD.png", alt="A layout with a header, three columns, and a footer.", width="800", height="607" %}
+
+A layout with a header, two sidebars, and a  footer, is known as the [holy grail layout](/patterns/layout/holy-grail/). There are many ways to mark up this content, including: 
+
+```html  
+<body>  
+    <header>Header</header>  
+    <nav>Nav</nav>  
+    <main>Content</main>  
+    <aside>Aside</aside>  
+    <footer>Footer</footer>  
+</body>  
+```
+
+If you are creating a blog, you might have a series of articles in `<main>`:
+
+```html  
+<body>  
+    <header>Header</header>  
+    <nav>Nav</nav>  
+    <main>   
+	<article>First post</article>  
+	<article>Second post</article>  
+    </main>  
+    <aside>Aside</aside>  
+    <footer>Footer</footer>  
+</body>  
+```
+
+Here, a `banner` and `contentinfo` are provided through a site `<header>` and `<footer>`.  The new elements added here include `<main>`, `<aside>`, and `<article>`; also, `<h1>` and `<nav>` that you used earlier, and `<section>`, which you haven't used yet.
```

Comment:
```suggestion
When employing semantic elements, browsers are able to create meaningful accessibility trees, enabling screen reader users to more easily navigate. Here, a `banner` and `contentinfo` are provided through a site `<header>` and `<footer>`.  The new elements added here include `<main>`, `<aside>`, and `<article>`; also, `<h1>` and `<nav>` that you used earlier, and `<section>`, which you haven't used yet.
```

Adding the first sentence explains why the image was included. It shows the banner and contentinfo.